plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -3484,6 +3483,31 @@ eosio::chain::backing_store_type read_only::get_backing_store() const {
 }
 
 } // namespace chain_apis
+
+fc::variant chain_plugin::get_entire_trx_trace(const std::variant<fc::exception_ptr, transaction_trace_ptr, std::shared_ptr<transaction>>& obj)const {
+
+    fc::variant pretty_output;
+    auto ro_api = get_read_only_api();
+    try {
+        if (std::holds_alternative<fc::exception_ptr>(obj)) {
+            abi_serializer::to_variant(std::get<fc::exception_ptr>(obj), pretty_output,
+                                       chain_apis::make_resolver(&ro_api, abi_serializer::create_yield_function(get_abi_serializer_max_time())),
+                                       abi_serializer::create_yield_function(get_abi_serializer_max_time()));
```

Comment:
No need for fc::exception to be converted by abi_serializer. Just assignment is all that is needed.

Reply:
Fixed.